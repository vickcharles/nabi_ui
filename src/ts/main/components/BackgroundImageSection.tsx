import * as React from 'react';
import { Button } from 'material-ui';

const BackgroundImageSection = () => {
  return (
    <section id="banner">
    <img src={require('../../../assets/images/nabiBannerImage.jpg')} alt=""/>
      <div className="container">
        <h1 className="nabi-font-montserrat nabi-text-semibold  nabi-font-jennasue"> 
        Share the love for music through education
        </h1>
          <p className=" nabi-text-mediumbold nabi-font-montserrat  ">
          Lear music with your city's most talented and qualified music instructors
          </p>
        <Button color="primary" variant="raised">
            FIND INSTRUCTOR
        </Button>
        <Button color="primary" variant="raised">
            START TEACHING
        </Button>
      </div>
    </section>
    );
};

export default BackgroundImageSection;