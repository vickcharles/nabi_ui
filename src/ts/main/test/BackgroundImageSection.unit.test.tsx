import * as React from 'react';
import BackgroundImageSection from '../components/BackgroundImageSection';
import { shallow } from 'enzyme';

describe('BackgroundImageSection', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(
      <BackgroundImageSection />
    );
  });

  it('Renders the component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
