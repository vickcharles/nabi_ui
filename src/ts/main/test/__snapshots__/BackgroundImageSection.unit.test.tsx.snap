// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImageSection Renders the component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BackgroundImageSection />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container nabi-text-center nabi-color-white"
      >
        <h1
          className="nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall"
        >
          Share the love for music through education
        </h1>
        <p
          className="nabi-margin-bottom-zero"
        >
          Learn music with your city's most talented and qualified music instructors
        </p>
        <WithStyles(Button)
          color="primary"
          variant="raised"
        >
          FIND INSTRUCTOR
        </WithStyles(Button)>
        <WithStyles(Button)
          color="primary"
          variant="raised"
        >
          START TEACHING
        </WithStyles(Button)>
      </div>,
      "id": "banner",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall"
          >
            Share the love for music through education
          </h1>,
          <p
            className="nabi-margin-bottom-zero"
          >
            Learn music with your city's most talented and qualified music instructors
          </p>,
          <WithStyles(Button)
            color="primary"
            variant="raised"
          >
            FIND INSTRUCTOR
          </WithStyles(Button)>,
          <WithStyles(Button)
            color="primary"
            variant="raised"
          >
            START TEACHING
          </WithStyles(Button)>,
        ],
        "className": "container nabi-text-center nabi-color-white",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Share the love for music through education",
            "className": "nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall",
          },
          "ref": null,
          "rendered": "Share the love for music through education",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Learn music with your city's most talented and qualified music instructors",
            "className": "nabi-margin-bottom-zero",
          },
          "ref": null,
          "rendered": "Learn music with your city's most talented and qualified music instructors",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "FIND INSTRUCTOR",
            "color": "primary",
            "variant": "raised",
          },
          "ref": null,
          "rendered": "FIND INSTRUCTOR",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "START TEACHING",
            "color": "primary",
            "variant": "raised",
          },
          "ref": null,
          "rendered": "START TEACHING",
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container nabi-text-center nabi-color-white"
        >
          <h1
            className="nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall"
          >
            Share the love for music through education
          </h1>
          <p
            className="nabi-margin-bottom-zero"
          >
            Learn music with your city's most talented and qualified music instructors
          </p>
          <WithStyles(Button)
            color="primary"
            variant="raised"
          >
            FIND INSTRUCTOR
          </WithStyles(Button)>
          <WithStyles(Button)
            color="primary"
            variant="raised"
          >
            START TEACHING
          </WithStyles(Button)>
        </div>,
        "id": "banner",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall"
            >
              Share the love for music through education
            </h1>,
            <p
              className="nabi-margin-bottom-zero"
            >
              Learn music with your city's most talented and qualified music instructors
            </p>,
            <WithStyles(Button)
              color="primary"
              variant="raised"
            >
              FIND INSTRUCTOR
            </WithStyles(Button)>,
            <WithStyles(Button)
              color="primary"
              variant="raised"
            >
              START TEACHING
            </WithStyles(Button)>,
          ],
          "className": "container nabi-text-center nabi-color-white",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Share the love for music through education",
              "className": "nabi-text-mediumbold nabi-jennasue-title nabi-color-white nabi-margin-bottom-xsmall",
            },
            "ref": null,
            "rendered": "Share the love for music through education",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Learn music with your city's most talented and qualified music instructors",
              "className": "nabi-margin-bottom-zero",
            },
            "ref": null,
            "rendered": "Learn music with your city's most talented and qualified music instructors",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "FIND INSTRUCTOR",
              "color": "primary",
              "variant": "raised",
            },
            "ref": null,
            "rendered": "FIND INSTRUCTOR",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "START TEACHING",
              "color": "primary",
              "variant": "raised",
            },
            "ref": null,
            "rendered": "START TEACHING",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
